src/main/java/Problem1.java

```diff
@@ -2,7 +2,11 @@
 
     // Do not change signature (function name, parameters)
     public static int binarySearch(int[] data, int target) {
-        // Add your solution
+        for (int i = 0; i < data.length; i++) {
+            if (data[i] == target) {
```

Comment:
-10: linear search instead of binary search

Reply:
using the wrong algorithm will cost you lots more points for a problem in future homework. when in doubt, check with the instructor.